cpp/src/aztec3/circuits/rollup/base/native_base_rollup_circuit.cpp

```diff
@@ -82,7 +82,9 @@ std::vector<NT::fr> calculate_contract_leaves(BaseRollupInputs baseRollupInputs)
 
             // @todo What to do about no contract deployments? Insert a zero, we talked with Mike.
             // For the nullifier member-ship, ignore when the leaf is zero.
```

Comment:
nit: explain the comment

Reply:
Updated the comment in https://github.com/AztecProtocol/aztec3-circuits/pull/114/commits/5ce273343e3b1eac65d345b59e6618e6d5aad703